metals/src/main/scala/scala/meta/metals/MetalsServices.scala

```diff
@@ -460,6 +481,7 @@ class MetalsServices(
             sbtServer = None
             showMessage.warn("Disconnected from sbt server")
         }
+        sbtExecWithMetalsPlugin("semanticdbEnable")
```

Comment:
Can we log something here? I tried this on a random project and it was silent for quite a while with no indication in the logs or elsewhere.

Reply:
Will do